config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -22,13 +22,6 @@ presubmits:
         - runner.sh
         - /workspace/scenarios/kubernetes_e2e.py
         args:
-        - --timeout=170
-        - --root=/go/src
-        - --repo=k8s.io/kubernetes=$(PULL_REFS)
```

Comment:
do we need to indicate somewhere to use feth the PULL_REFS?

Reply:
pod utils takes care of this